RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -66,24 +84,16 @@ public bool Generate(NuGetReference analyzeRef, string sqaleFilePath)
 
                 string rulesFilePath = Path.Combine(outputDirectory, "rules.xml");
 
+                // TODO: we shouldn't try to work out where the content files have been installed by NuGet.
+                // Instead, we should use the methods on IPackage to locate the assemblies e.g. Package.GetFiles()
```

Comment:
FYI I have fixed this TODO in another set of commits. I'll send them out for review separately.